webpack/scrambleLogo.js

```diff
@@ -0,0 +1,22 @@
+const characters =
+  "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+
+const letter = [];
+const scramble = [];
+
+for (let i = 0; i < 5; i++) {
+  letter[i] = document.getElementById("splash--logo-letter-" + i);
+  scramble[i] = setInterval(function() {
+    letter[i].innerHTML = characters.charAt(
+      Math.floor(Math.random() * characters.length)
+    );
+  }, 80);
+}
+
+const logo = "IMEsec";
+for (let i = 0; i < 5; i++)
+  setTimeout(function() {
+    letter[i].innerHTML = logo[i];
+    clearInterval(scramble[i]);
+    letter[i].classList.add("error");
+  }, 2000 + 250 * i);
```

Comment:
how could we make the animation not run every single time on the end user? That'd be annoying to cope with. Maybe we can use a small LocalStorage variable? We can make the animation run again when the page is refreshed. There must be a smart solution

Reply:
Yes... I'll search for alternatives

Reply:
But first, I'm trying to substitute this mess with the lottie lib